feat(rfs): add datasource to get the list of RFS template versions (#9385)
huaweicloud/terraform-provider-huaweicloud: docs/data-sources/rfs_template_versions.md

---
subcategory: "Resource Formation (RFS)"
layout: "huaweicloud"
page_title: "HuaweiCloud: huaweicloud_rfs_template_versions"
description: |-

>   Use this data source to get the list of RFS template versions.

Reviewer: ```suggestion
  Use this data source to get all versions of a RFS template.
```

---

# huaweicloud_rfs_template_versions

> Use this data source to get the list of RFS template versions.

Reviewer: ```suggestion
Use this data source to get all versions of a RFS template.
```

## Example Usage

```hcl
variable "template_name" {}

data "huaweicloud_rfs_template_versions" "test" {
  template_name = var.template_name
}
```

## Argument Reference

The following arguments are supported:

* `region` - (Optional, String) Specifies the region in which to query the resource.
  If omitted, the provider-level region will be used.

* `template_name` - (Required, String) Specifies the template name.

* `template_id` - (Optional, String) Specifies the template ID.

## Attribute Reference

In addition to all arguments above, the following attributes are exported:

* `id` - The data source ID.

> * `versions` - The list of RFS templates.

Reviewer: ```suggestion
* `versions` - All template versions of the templete.
```
